integration-tests/bot/join_test.py

```diff
@@ -0,0 +1,331 @@
+#!/usr/bin/python3
+import docker
+import json
+import requests
+import datetime
+import time
+import jsonrpcclient
+from chainrpc import RPC, Blockchain
+from decouple import config
+from chainbot import SigningKey
+CURRENT_HASH = config('CURRENT_HASH', '')
+class Program :
+    def __init__(self) :
+        self.rpc = RPC()
+        self.blockchain = Blockchain()
+        # wallet a
+        self.node0_address = ""
+        self.node0_address1 = ""
+        self.node0_transfer_address = ""
+        self.node0_mnemonics= ""
+
+        # wallet b
+        self.node1_address = ""
+        self.node1_address1= ""
+        self.node1_transfer_address = ""
+        self.node1_mnemonics=""
+
+        # wallet b
+        self.node2_address = ""
+        self.node2_mnemonics=""
+
+
+
```

Comment:
Why so many blank lines ;D, you can run a python linter tools like flake8.

Reply:
ok, i'll remove it